### PR TITLE
894 create event

### PR DESCRIPTION
**Pull request by ippudo**
using API to detect the max transaction fee for Create Event ( #892 )
remove bot shown on "approve" at txconfirmdialog  ( #894 )
correct css style for button on oracle pages ( #847 )

### CODE REVIEW

File: src/scenes/CreateEvent/store.js [hunk 1]
```diff
@@ -171,6 +177,32 @@ export default class CreateEventStore {
...
+    try {
+      const { data: { result } } = await axios.post(
+        Routes.api.transactionCost,
+        {
+          type: TransactionType.APPROVE_CREATE_EVENT,
+          token: Token.BOT,
+          amount: decimalToSatoshi(this.escrowAmount),
+          optionIdx: undefined,
+          topicAddress: undefined,
+          oracleAddress: undefined,
```
Comment (dwalintukan) [inline src/scenes/CreateEvent/store.js]: Don't need to add any `undefined` keys, just leave them out. By default, every var not explicitly defined is `undefined`.

File: src/scenes/Event/store.js [hunk 2]
```diff
@@ -118,6 +118,30 @@ export default class EventStore {
     this.setReactions();
   }
 
+  @action
+  async initTxfees() {
```
Comment (dwalintukan) [inline src/scenes/Event/store.js]: Why do you need this? Tx cost should be calculated when the user does the action. And this one is assuming we are doing a `bet`, but the EventStore is used for all types of txs. 

It is also unused, remove it.

File: src/scenes/CreateEvent/store.js [hunk 3]
```diff
@@ -171,6 +177,32 @@ export default class CreateEventStore {
...
+          senderAddress: this.app.wallet.lastUsedAddress,
+        }
+      );
+      const txFees = _.map(result, (item) => new TransactionCost(item));
+      runInAction(() => {
```
Comment (dwalintukan) [inline src/scenes/CreateEvent/store.js]: Don't believe you need to wrap this in an action. It's already in an action.

File: src/scenes/CreateEvent/store.js [hunk 4]
```diff
@@ -171,6 +177,32 @@ export default class CreateEventStore {
...
+        this.txFees = txFees;
+      });
+    } catch (error) {
+      runInAction(() => {
+        this.app.ui.setError(error.message, Routes.api.transactionCost);
```
Comment (dwalintukan) [inline src/scenes/CreateEvent/store.js]: same, prob dont need to wrap in `runInAction`

File: src/scenes/CreateEvent/store.js [hunk 5]
```diff
@@ -171,6 +177,32 @@ export default class CreateEventStore {
     );
   }
 
+  @action
+  onEnterDialog = async () => {
+    try {
+      const { data: { result } } = await axios.post(
```
Comment (dwalintukan) [inline src/scenes/CreateEvent/store.js]: Why is this logic duplicated in two places? it is the same logic as `prepareToCreateEvent()`. We don't need to call this logic twice.

**Comment by dwalintukan** [comment]
@ippudo Also, for this #847 you forgot to handle number 1.

File: src/scenes/Event/components/Button.js [hunk 6]
```diff
@@ -6,8 +6,10 @@ export const Button = withStyles(styles, { withTheme: true })(({ classes, ...pro
   <_Button
     fullWidth
     size='large'
+    color='default'
```
Comment (dwalintukan) [inline src/scenes/Event/components/Button.js]: why default? we normally use `primary` to use the defined primary color from the theme.

File: src/scenes/Event/components/Button.js [hunk 7]
```diff
@@ -6,8 +6,10 @@ export const Button = withStyles(styles, { withTheme: true })(({ classes, ...pro
...
+    variant='outlined'
```
Comment (dwalintukan) [inline src/scenes/Event/components/Button.js]: whats the difference of `outlined` and `raised`?

File: src/scenes/Event/components/styles.js [hunk 8]
```diff
@@ -1,11 +1,8 @@
 const styles = (theme) => ({
   oracleButton: {
+    marginTop: theme.padding.md.px,
+    background: theme.palette.primary.main,
     color: theme.palette.primary.contrastText,
```
Comment (dwalintukan) [inline src/scenes/Event/components/styles.js]: if you set `color='primary'` in the button itself, you shouldn't have to define the colors. look at how we use all the other buttons, eg. `Wallet/index.js L302`. when it comes to UI elements, try to be consistent with the same implementation as other places.

**Comment by dwalintukan** [comment]
@ippudo Don't forget to handle number 1 in #847.

**Comment by ippudo** [comment]
Some UI changes that not mentioned at the issues are expected with this pr.

**Pull request merged**